### PR TITLE
feat(properties-item): update validationSchema on changes in head

### PR DESCRIPTION
**Pull request by RobinStut**

### CODE REVIEW

File: src/heads-up/components/properties-item.vue [hunk 1]
```diff
@@ -147,14 +152,20 @@ export default {
...
+  methods:{
+    validateSchema(){
+      const schemaTemplate = {
+        schema: this.schema,
+        key: this.keyName,
+        value: this.value,
+        attrs: this.attrs
+      }
+      
+      this.errors = validateSchema(schemaTemplate)
```
Comment (RobinStut) [inline src/heads-up/components/properties-item.vue:166]: I put this schemaTemplate in a const to reuse later on. I am planning to create multiple validation functions like rule 166

File: src/heads-up/components/properties-item.vue [hunk 2]
```diff
@@ -134,6 +134,11 @@ export default {
       return null;
     }
   },
+  watch:{
+    'head'(){
```
Comment (Jamerrone) [inline src/heads-up/components/properties-item.vue]: `'head'() {`

File: src/heads-up/components/properties-item.vue [hunk 3]
```diff
@@ -147,14 +152,20 @@ export default {
...
-      schema: this.schema,
-      key: this.keyName,
-      value: this.value,
-      attrs: this.attrs,
-    })
-  }
+    this.validateSchema()
+  },
+  methods:{
+    validateSchema(){
```
Comment (Jamerrone) [inline src/heads-up/components/properties-item.vue]: `validateSchema() {`

**Review by Jamerrone — CHANGES_REQUESTED** [review]
You are missing 2 spaces.

**Review by Jamerrone — APPROVED** [review]

**Pull request merged**